src/api/measurement.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package api;
+
+message Measurement {
+        optional uint32 motion = 101;
+	optional uint32 water = 102;
+	optional uint32 temperature = 103;
+        optional float humidity = 104;
```

Comment:
Spacing.

Also, starts fields from 1. Temperature is a `float` as well. Make two versions; `temperature_celcius` and `temperature_fahrenheit`